src/Equations/RunawayFluid.cpp

```diff
@@ -866,6 +866,28 @@ real_t* RunawayFluid::evaluatePartialContributionAvalancheGrowthRate(len_t deriv
     return dGamma;
 }
 
+/**
+ * Same as the function above, but for the formula published by Hesslow et al NF 2019
```

Comment:
Instead of ``function above``, you could perhaps give the functions name explicitly? In case code moves around in the future